proof/refine/ARM/TcbAcc_R.thy

```diff
@@ -548,13 +548,17 @@ lemma assert_get_tcb_corres:
   apply assumption
   done
 
+schematic_goal threadGet_getObject:
+  "threadGet f t = ?x"
```

Comment:
Please spell out the `?x`, otherwise if we later change any definition below this we won't know if we need to make a change here or not.